### PR TITLE
feat: Extension spec yml support i18n

### PR DESCRIPTION
**Pull request by shuofan**
#### What this PR does / why we need it:
Extension spec yml i18n set up

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @Effet 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     Extension spec yml  support i18n        |
| 🇨🇳 中文    |    拓展 spec.yml 支持国际化          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkg/expression/expression_reconcile.go [hunk 1]
```diff
@@ -43,6 +43,7 @@ const (
 	Configs      = "configs"
 	Base64Decode = "base64-decode"
 	TriggerLabel = "triggers"
+	I18n         = "i18n"
```
Comment (Effet) [inline pkg/expression/expression_reconcile.go:46]: Where is it used?

**Review by Effet — DISMISSED** [review]

File: pkg/expression/expression_reconcile.go (hunk 1 above)
Comment (shuofan) [reply pkg/expression/expression_reconcile.go:46]: when replacing this prefix in spec yml

**Review by sfwn — APPROVED** [review]

File: modules/dicehub/service/extension/extension.go [hunk 2]
```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/erda-project/erda/modules/dicehub/conf"
 	"github.com/erda-project/erda/modules/dicehub/dbclient"
 	"github.com/erda-project/erda/modules/dicehub/service/apierrors"
+	"github.com/erda-project/erda/modules/pipeline/pexpr"
```
Comment (sfwn) [inline modules/dicehub/service/extension/extension.go:36]: move to pkg/parser/pexpr later

**Pull request merged**